www/templates/user-no-berkeley-edu-emails

```diff
@@ -7,5 +7,10 @@ addresses @<yourgroup>.berkeley.edu as you'd like (you can create them
 for club members, officers, etc.). This is open to all RSOs. We can also
 provide just mail hosting (without a website) if you prefer.
 
-If you're only interested in an @berkeley.edu email, you'll want to
-contact someone with the university instead.
```

Comment:
Overall I think this is an improvement, but I think the old wording was slightly better at making it clear that we have nothing to do with this process. Now that we're telling them the exact instructions to follow, I worry people might misunderstand our role in the process. We wouldn't want them to think this is some kind of service we provide (or can provide support for).

It might be better to say something like

> If you're only interested in an @berkeley.edu email, you'll need to work with the university to get one. They call these "Special Purpose Accounts" (SPAs); you can find information at https://idc.berkeley.edu/spa/ about them.
>
> The process requires you to have a faculty or staff member create a new SPA and then share the account with members of your group. The university provides instructions for that at https://calnetweb.berkeley.edu/calnet-departments/special-purpose-accounts-spa

Something like this provides the same information, but imo makes it more obvious to people skimming the email that it's not an OCF service.

Reply:
Agreed, I think that's a much better way to phrase it. I did think that it might be a concern that we would be doing the work for them to obtain an address, so I'll change the wording. Thanks for the suggestion!

Reply:
One problem with this text is that https://idc.berkeley.edu/spa/ isn't available for students at all, they just get a message about not being allowed access, so we should probably just tell them to look at the SPA docs and have a staff/faculty member do the whole setup, something like that anyway.